src/guacd/daemon.c

```diff
@@ -275,6 +281,17 @@ int main(int argc, char* argv[]) {
     /* General */
     int retval;
 
+#ifdef PTHREAD_STACK_SIZE
+    /* Set default stack size */
+    pthread_attr_t default_pthread_attr;
+    if (pthread_attr_init(&default_pthread_attr) ||
+        pthread_attr_setstacksize(&default_pthread_attr, PTHREAD_STACK_SIZE) ||
+        pthread_setattr_default_np(&default_pthread_attr)) {
+       fprintf(stderr, "Couldn't set requested pthread stack size of %d\n", PTHREAD_STACK_SIZE);
+       exit(EXIT_FAILURE);
```

Comment:
Please use 4-space indents per level. See: http://guacamole.apache.org/guac-style/#general-style

Reply:
Got it! Will fix.